classes/DirtSoundLibrary.sc

```diff
@@ -284,6 +332,11 @@ DirtSoundLibrary {
 			stretchInstrument: this.stretchInstrumentForBuffer(buffer),
 			bufNumFrames: buffer.numFrames,
 			bufNumChannels: buffer.numChannels,
+			baseFreqToFreqRatio: {
+				if(~tune.notNil && { ~tune > 0 }) {
```

Comment:
for efficiency use `and:` instead of `&`.

Reply:
the enclosing function is called from `addBuffer` but `~tune` doesnt exist yet at this point, it is a patternable argument (that is why i have the wrapping function). but thinking about it some more, the whole ratio is only relevant to the `sound` module, so actually i think it makes more sense to do the math in the module code instead..

youre right about the ambiguity, but `tuneToMetaData` does feel quite long compared to other argument names. how about `metatune`? (imagining a naming convention `metafoo` for "do foo using metadata")